azurePipelineTemplates/run_scenarios.yml

```diff
@@ -0,0 +1,117 @@
+        & "./$build_name" "make" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}"
+
+        echo "Copy a test file from local to the created container"
+        echo "Hello, Azure!" > myfile.txt
+        & "./$build_name" "cp" "myfile.txt" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}"
+
+        & "./$build_name" "cp" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}/myfile.txt" "--from-to=BlobPipe" "--check-md5" "FailIfDifferentOrMissing" > filename
+        
+        if ($LASTEXITCODE -eq 0) {
+            Write-Output "Successfully executed the scenario."
+        } else {
+            Write-Error "AzCopy command failed with exit code $LASTEXITCODE."
+        }
+
+        # Perform cleanup
+        echo "Perform cleanup"
+        echo "Remove the test file from the container"
+        & "./$build_name" "rm" "https://${{ parameters.storage_account_name }}.blob.core.windows.net/${{ parameters.container_name }}/myfile.txt"
+        echo "Remove the container from the storage account"
+        & az storage container delete --name ${{ parameters.container_name }} --account-name ${{ parameters.storage_account_name }}
+
+        ######################################################## Scenario 2 ########################################################
+
+        # Test copying from Object Replication Policy source container to destination container
+        # Assign an object replication policy to a storage account. 
+        # Perform azcopy copy operation to copy the blob to another container.
+
+        echo "Scenario 2: Test copying from Object Replication Policy source container to destination container"
+
+        echo "Create test file myfile.txt and copy the file to the source container srcorpcontainer under storage account azcopyorpsrcaccount having the ORP policy"
+        echo "Hello, Azure!" > myfile.txt
+        & "./$build_name" "cp" "myfile.txt" "https://${{ parameters.orp_source_account_name }}.blob.core.windows.net/srcorpcontainer"
+
+        echo "Copy the test file from the source container to the destination container testcontainer under storage account azcopyorpdestaccount"
+        & "./$build_name" "cp" "https://${{ parameters.orp_source_account_name }}.blob.core.windows.net/${{parameters.orp_source_container_name}}" "https://${{ parameters.orp_destination_account_name }}.blob.core.windows.net/${{parameters.destination_container_name}}" --recursive
+
+        if ($LASTEXITCODE -eq 0) {
+            Write-Output "Successfully executed the scenario."
+        } else {
+            Write-Error "AzCopy command failed with exit code $LASTEXITCODE."
+        }
+
+        # Perform cleanup
+        echo "Perform cleanup"
+        echo "Remove the test file from the destination container"
```

Comment:
Should we also do cleanup for source ORP account?

Reply:
No we have to create this src ORP account manually. Just cleaning up the destination container would work.

Reply:
I meant deleting the test file from the source container